client/src/pages/Bookings/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,34 @@
+import { LocalizationProvider, PickersDay, StaticDatePicker } from '@mui/lab';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import { TextField } from '@mui/material';
+import { Request } from '../../../interface/RequestApiData';
+
+interface Props {
+  requests: Request[];
+}
+
+export default function Calendar({ requests }: Props): JSX.Element {
+  const initialDate = new Date();
+  const highlightedDays = requests.map((x) => new Date(x.start).toDateString());
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
We can move the `LocalizationProvider` to our App.tsx so that it can be used throughout our app